tests/test_gc.rs

```diff
@@ -81,52 +81,6 @@ fn data_is_dropped() {
     assert!(drop_called2.load(Ordering::Relaxed));
 }
 
-#[pyclass]
-struct ClassWithDrop {}
-
-impl Drop for ClassWithDrop {
-    fn drop(&mut self) {
-        unsafe {
-            let py = Python::assume_gil_acquired();
-
-            let _empty1: Py<PyTuple> = FromPy::from_py(PyTuple::empty(py), py);
```

Comment:
So what is this test for?
@konstin do you know?

Reply:
Afaik they are stored in the pointer list belonging to the GIL wrapper, so we can observe in `create_pointers_in_drop` that they are only dropped when our GIL wrapper cleans up .